src/function_lua.c

```diff
@@ -224,56 +231,237 @@ static void luaEngineFreeFunction(void *engine_ctx, void *compiled_function) {
     zfree(f_ctx);
 }
 
-static int luaRegisterFunction(lua_State *lua) {
-    int argc = lua_gettop(lua);
-    if (argc < 2 || argc > 3) {
-        luaPushError(lua, "wrong number of arguments to redis.register_function");
-        return luaRaiseError(lua);
+static void luaRegisterFunctionArgsInitialize(registerFunctionArgs *register_f_args,
+                                              sds name,
+                                              sds desc,
+                                              luaFunctionCtx *lua_f_ctx,
+                                              uint64_t flags)
```

Comment:
i'd rather not indent argument to the function name.
- it means that when the function is renamed the indentation gets broken
- might not leave a lot of room for long var names or comments,.

Reply:
Fixed